action.js

```diff
@@ -328,17 +346,20 @@ async function createTask(
         taskOpts.memberships = [{ project: projectId, section: sectionId }];
     }
 
+    const body = { data: taskOpts };
+    const opts = {};
+
     console.log(`creating new task with options:='${JSON.stringify(taskOpts)}'`);
     let createdTaskId = '0';
     try {
-        await client.tasks.createTask(taskOpts).then((result) => {
-            createdTaskId = result.gid;
+        await client.tasks.createTask(body, opts).then((result) => {
+            createdTaskId = result.data.gid;
             console.log('task created', createdTaskId);
             core.setOutput('taskId', createdTaskId);
             core.setOutput('duplicate', false);
         });
     } catch (error) {
-        console.error('rejecting promise', JSON.stringify(error));
+        console.error(error.response.body);
```

Comment:
I don't think that's right - we're printing the JS error here not the HTTP response